tests/matrix.cpp

```diff
@@ -76,3 +75,10 @@ BOOST_AUTO_TEST_CASE( lookat )
     BOOST_CHECK_EQUAL( lookAt, newLookAt );
     BOOST_CHECK_EQUAL( up, newUp );
 }
+
+// Verify code by instantiating some templates:
+template class vmml::Matrix< 1, 1, float >;
+template class vmml::Matrix< 2, 2, double >;
+template class vmml::Matrix< 3, 3, short >;
+template class vmml::Matrix< 3, 4, int >;
+//template class vmml::Matrix< 4, 4, unsigned >;
```

Comment:
Some maths operations with unsigned are not commutative on type T. Therefore other than keeping data most maths operations will be ill formed with operations on commutative properties. In short a - b != b - a